src/transport_performance/utils/defence.py

```diff
@@ -120,7 +126,12 @@ def _is_expected_filetype(pth, param_nm, check_existing=True, exp_ext=".zip"):
     _, ext = os.path.splitext(pth)
     if check_existing and not os.path.exists(pth):
         raise FileExistsError(f"{pth} not found on file.")
-    if ext != exp_ext:
+    if isinstance(exp_ext, list):
+        is_correct = ext in exp_ext
+    elif isinstance(exp_ext, str):
+        is_correct = ext == exp_ext
```

Comment:
It could be worth lowering both ext and exp_ext here. Each item in exp_ext would need to be lowered in the case of a list. This could be a small QOL improvement